docs/style.md

```diff
@@ -0,0 +1,106 @@
+# Overview
+
+All applicable styles should be colocated with their corresponding JavaScript component using [Aphrodite](https://github.com/Khan/aphrodite).
+
+## Legacy
+
+All legacy styles are processed with [Less](http://lesscss.org/) and can be found in the /less directory. These should still be maintained but all future CSS should be written in JavaScript and kept inside the appropriate component file.
+
+## Example
+
+Here is an example of a React component styled with Aphrodite.
+
+Aphrodite will automatically create a single `<style>` tag in the document `<head>` to put its generated styles in.
+
+```jsx
+const React = require('react')
+const ImmutableComponent = require('./immutableComponent')
+const Aphrodite = require('aphrodite')
+const StyleSheet = Aphrodite.StyleSheet
+const css = Aphrodite.css
+
+class Button extends ImmutableComponent {
+  render () {
+    return <span className={css(styles.browserButton)} 
+      disabled={this.props.disabled}
+      data-l10n-id={this.props.l10nId}
+      data-button-value={this.props.dataButtonValue}
+      onClick={this.props.onClick} />
+  }
+}
+
+const styles = StyleSheet.create({
+  browserButton: {
+    cursor: 'default',
+    display: 'inline-block',
+    lineHeight: '25px',
+    width: '25px',
+    height: '25px',
```

Comment:
Can you make a recommendation on units as well as part of this documentation? Should we avoid px for example on new stuff or is it ok?  We seem to have some support issues along the lines of things look tiny, maybe due to dpi? Not sure but just wondering if this type of thing could be handled at the same time as we do a new style system.  Now would be a great time to figure that type o thing out and make sure new things use the new system.